go/ephemeral/lib.go

```diff
@@ -357,7 +377,7 @@ func (e *EKLib) getOrCreateLatestTeamEKInner(ctx context.Context, teamID keybase
 	cacheKey := e.cacheKey(teamID)
 	val, ok := e.teamEKGenCache.Get(cacheKey)
 	if ok {
-		if cacheEntry, valid := e.isEntryValid(val); valid {
+		if cacheEntry, expired := e.isEntryExpired(val); !expired || cacheEntry.CreationInProgress {
```

Comment:
Could this cause trouble if background key creation fails? I'm worried we could be in a situation where the `CreationInProgress` flag is set, and it never gets unset, and so the `expired` bool never gets respected.

Reply:
we clear the cache on errors: https://github.com/keybase/client/pull/12553/files#diff-6dc458a542227a8fc53f68de4223c8baR430